action.yml

```diff
@@ -1,16 +1,63 @@
-name: Actions Ecosystem Action Push Tag
-description: Push a Git tag.
-author: Actions Ecosystem
+name: Push Any Git Tag
+description: based on Actions Ecosystem Push Tag - Fixed with additional features.
+author: Actions Ecosystem & theJeff77
+
 inputs:
   tag:
     description: A Git tag name.
+    type: string
     required: true
   message:
     description: A message for the Git tag.
+    type: string
     required: false
+  ref:
+    description: Commit SHA or ref (tag) to target for the tag.
+    type: string
+    required: false
+  force:
+    description: Whether or not to force push (overwrite) the tag if it already exists. Useful for walking tags.
```

Comment:
This descriptions are not available for me when I read the docs or trying to see it will work for me